### PR TITLE
Dont ignore missing undefined keys in jinja2 templates

### PR DESCRIPTION
**Pull request by eddwardo**
To keep templates clean, make sure that all none of the keys are
undentified. Log a helpful message upon the error and exit with 1 code.
while rendering: webapp/my.cnf ('secret' is undefined)

### CODE REVIEW

File: confgen/cli.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from logic import ConfGen
 from . import __version__
 
-logging.getLogger('confgen').addHandler(logging.StreamHandler())
+logging.basicConfig(format='[%(levelname)s] %(message)s')
```
Comment (io41) [inline confgen/cli.py]: What's this? Why no StreamHandler? Change seems unrelated to the rest

Comment (eddwardo) [inline confgen/cli.py]: you get a StreamHandler by default and I wanted a nicer formatting as well.

Comment (io41) [inline confgen/cli.py]: 👍
